core/templates/pages/exploration-player-page/exploration-player-page.component.spec.ts

```diff
@@ -18,11 +18,13 @@
 
 // TODO(#7222): Remove the following block of unnnecessary imports once
 // App.ts is upgraded to Angular 8.
-import 'mousetrap';
 
 import { UpgradedServices } from 'services/UpgradedServices';
 // ^^^ This block is to be removed.
 
+import { KeyboardShortcutService } from 'services/keyboard-shortcut.service';
+import { TestBed } from '@angular/core/testing';
```

Comment:
Import angular lib on top, then a newline, and finally the oppia libs. (You can check `core/templates/components/button-directives/exploration-embed-button-modal.component.spec.ts` for ref!)